src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/GeneralTests.cs

```diff
@@ -287,6 +287,7 @@ public static void RoundTrip_ExplicitSki()
         }
 
         [Fact]
+        [OuterLoop(/* Leaks key on disk if interrupted */)]
         public static void FromManagedPal_CompatWithOctetStringWrappedContents_Decrypt()
```

Comment:
Note for self: need to make this test only run on platforms that use the managed PAL.